database/dynamodb.go

```diff
@@ -38,44 +38,42 @@ func getAWSCredentials() *AWSCredentials {
 	if !found {
 		log.Println("AWS region not stored, please store it.")
 
-		// Here the error is passed for debugging to the user we show "Something went wrong"
-		utils.ServerError(errors.New("AWS region not stored in ENV var")) 
+		utils.ServerError(errors.New("AWS region not stored in ENV var"))
```

Comment:
Why has this changed there is no difference

Reply:
He has removed the extra line left after the comment got removed.